docs/pages/branding/about.tsx

```diff
@@ -530,15 +530,6 @@ function AboutContent() {
               Join the team
             </Button>
           </div>
-          <Box display={{ xs: 'none', sm: 'block' }}>
-            <img
-              width="130"
-              height="124"
-              src="/static/branding/pricing/early-bird.svg"
```

Comment:
@siriwatknp  We could probably delete this image from the repo. Didn't do it so I don't break stuff 😬 Left it to you.

Reply:
Let's leave it there, we will probably have a cleanup PR soon. (remove unused code from previous rebranding)